oat-sa/e2e-runner
# Feature/ADF-1417/Upgrade to Node 18

### Related to: [ADF-1417](https://oat-sa.atlassian.net/browse/ADF-1417)

### Summary

Align the dependencies for ESLint in order to prepare the support for Node 18.

**Note:** The repo is not yet compatible with Node 18, we need to update or replace `cypress-plugin-snapshots` first.

### Details

To properly support Node 18, the dev dependencies had to be updated and aligned.

**Note:** The lock file was generated with `npm@6` so it would also work seamlessly with Node 14.

### How to test
- check the report on GitHub
- make sure to have Node 18 installed (you can use [nvm.sh](https://github.com/nvm-sh/nvm) for managing multiple versions)
- install the package: `npm i`
- test the linter: `npm run lint`


[ADF-1417]: https://oat-sa.atlassian.net/browse/ADF-1417?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
